crates/metastore/src/storage/lease.rs

```diff
@@ -242,7 +242,7 @@ impl LeaseRenewer {
             })?;
 
             // Some other process has the lease.
-            if expires_at > now {
+            if expires_at > now && held_by != self.process_id {
```

Comment:
i presume that `self.process_id` is a UUID generated on process start?

Reply:
yep, see https://github.com/glaredb/glaredb/blob/30ff37ca38d32a0084c6c620b6a333bc81274d5b/crates/metastore/src/srv.rs#L40-L49